src/p2p/net_node.h

```diff
@@ -226,7 +226,7 @@ namespace nodetool
         typename request_struct::response resp = AUTO_VAL_INIT(resp);
         bool r = epee::net_utils::invoke_http_json(m_supernode_uri + uri,
                                                    req, resp, m_supernode_client,
-                                                   std::chrono::milliseconds(SUPERNODE_HTTP_TIMEOUT_MILLIS), "POST");
+                                                   std::chrono::milliseconds(size_t(SUPERNODE_HTTP_TIMEOUT_MILLIS)), "POST");
```

Comment:
Perhaps you should actually fix the problem rather than hacking around it: you have a static constexpr variable without a definition, which isn't valid (until C++17).

Reply:
@jagerman Thank you for pointing us to this issue. Please feel free to provide correct fix as a separate PR.

Reply:
https://www.learncpp.com/cpp-tutorial/811-static-member-variables/